### DISCUSSION TITLE
Is it possible to display the file's metadata in "rich row format" when viewing the file?

### DISCUSSION THREAD

**Discussion by johtso** [RafaelGB/obsidian-db-folder#715] [Q&A]
Currently when I view the file related to a row, it's just raw frontmatter.

Is there a way to display the current row so I can edit it without having to go back to my database file?

**Comment by RafaelGB** [comment]
I don't quite understand at what point you are looking to have that format.

The plugin just can edit the information inside the database view. An editor UI inside the file could be a long term feature with a modal helper
